samples/package.json

```diff
@@ -14,9 +14,9 @@
     "test": "mocha system-test --timeout 600000"
   },
   "dependencies": {
+    "@google-cloud/pubsub": "^2.10.0",
     "@opentelemetry/api": "^0.11.0",
-    "@opentelemetry/tracing": "^0.11.0",
-    "@google-cloud/pubsub": "^2.10.0"
+    "@opentelemetry/tracing": "^0.11.0"
```

Comment:
I don't know why npm keeps swapping these around, but I finally just agreed to it :P

Reply:
npm sometimes likes to put things in alphabetical order for you :)